docs/development/howto.rst

```diff
@@ -152,18 +152,18 @@ weren't available for testing on travis-ci.
 Wipe readthedocs
 ----------------
 
-As described `here <http://read-the-docs.readthedocs.org/en/latest/builds.html#deleting-a-stale-or-broken-build-environment>`__,
+As described `here <http://read-the-docs.readthedocs.io/en/latest/builds.html#deleting-a-stale-or-broken-build-environment>`__,
 if the docs on readthedocs show old stuff, you need to first log in `here <https://readthedocs.org/accounts/login/>`__
 and then wipe it to create a fresh / clean version by hitting this URL::
 
-   http://readthedocs.org/wipe/gammapy/latest/
+   http://readthedocs.io/wipe/gammapy/latest/
```

Comment:
https://readthedocs.org/wipe/gammapy/latest/ does work if you're an admin on RTD for Gammapy and logged in, so that's the correct one to put here in the docs.